staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/predicates/object/matcher_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package object
+package object_test
```

Comment:
The package should be `object`?

Reply:
`object_test` is a way of avoiding cyclical imports. In this case, the object matcher testing imports the webhook package (which now imports the interface definition), so putting the test code in `object_test` breaks that cycle.

I can't find much in the official go docs, but here's a blog post about it:

https://blog.jpalardy.com/posts/which-package-name-for-go-tests/

Another option would be to move the interfaces into a separate `types` package, but I figured defining the interface next to the thing that consumes it would be easier to grok.

Reply:
Thanks for sharing! Didn't know it before^^ LGTM